include/dem/locate_ghost_particles.h

```diff
@@ -0,0 +1,53 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2020 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ * Author: Shahab Golshan, Polytechnique Montreal, 2019
+ */
+
+#include <dem/update_ghost_iterator_pp_contact_container.h>
+#include <dem/update_ghost_particle_container.h>
+
+using namespace dealii;
+
+#ifndef LOCATEGHOSTPARTICLES_H_
+#  define LOCATEGHOSTPARTICLES_H_
```

Comment:
Don't caps lock for the include guards. We don,t do that in both Lethe and deal.II
You could just write locate_ghost_particles_h